lighthouse-extension/app/src/popup.js

```diff
@@ -122,6 +126,21 @@ document.addEventListener('DOMContentLoaded', _ => {
     return listItem;
   }
 
+  function showError(err, message, includeReportLink) {
+    sourcemapStacktrace.mapStackTrace(err.stack, stackWithSourcemaps => {
```

Comment:
can we do a try/catch around the mapStackTrace call? I want to make sure we gracefully degrade and still show the error reporting stuff even if the sourcemap resolution fails.